manifests/install.pp

```diff
@@ -3,7 +3,7 @@
   $ensure  = 'latest',
   ) {
 
-  package { 'scout':
+  package { '^scout':
```

Comment:
It's fixed in 3.7.0 https://github.com/puppetlabs/puppet/commit/913658327eb1673e1ed9844b6b3bff235f978160